src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -0,0 +1,401 @@
+﻿namespace Equinox.MessageDb
+
+open System
+open System.Text.Json
+open System.Threading.Tasks
+open Equinox.Core
+open FSharp.Control
+open FsCodec
+open Npgsql
+open Serilog
+
+type EventBody = JsonElement
+type EventData = IEventData<JsonElement>
+type IEventStoreConnection = MessageDbClient
```

Comment:
not sure this is a good alias - makes it harder to read this, and that name is not stable across ESDB impls anyway